File: src/main/java/com/saybetter/domain/solution/application/impl/SolutionSymbolService.java
package com.saybetter.domain.solution.application.impl;

import java.util.List;

import org.springframework.stereotype.Service;

import com.saybetter.domain.solution.dao.repository.SolutionSymbolReadRepository;
import com.saybetter.domain.solution.dao.repository.SolutionSymbolWriteRepository;
import com.saybetter.domain.solution.domain.SolutionSymbol;

import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;

@Slf4j
@Service
@RequiredArgsConstructor
public class SolutionSymbolService {

	private final SolutionSymbolReadRepository solutionSymbolReadRepository;
	private final SolutionSymbolWriteRepository solutionSymbolWriteRepository;

	public void createSolutionSymbols(List<SolutionSymbol> solutionSymbols) {
		solutionSymbolWriteRepository.saveAll(solutionSymbols);
	}

}

File: core/core-api/src/main/java/io/say/better/domain/solution/application/impl/SolutionSymbolService.kt
package io.say.better.domain.solution.application.impl;

import java.util.List;

import org.springframework.stereotype.Service;

import io.say.better.storage.mysql.dao.repository.SolutionSymbolReadRepository;
import io.say.better.storage.mysql.dao.repository.SolutionSymbolWriteRepository;
import io.say.better.storage.mysql.domain.entity.SolutionSymbol;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;

@Slf4j
@Service
@RequiredArgsConstructor
public class SolutionSymbolService {

	private final SolutionSymbolReadRepository solutionSymbolReadRepository;
	private final SolutionSymbolWriteRepository solutionSymbolWriteRepository;

	public void createSolutionSymbols(List<SolutionSymbol> solutionSymbols) {
		solutionSymbolWriteRepository.saveAll(solutionSymbols);
	}

}
